ddtrace/tracer/sampler.go

```diff
@@ -384,6 +396,28 @@ func (sr *SamplingRule) match(s *span) bool {
 	return true
 }
 
+// MarshalJSON implements the json.Marshaler interface.
+func (sr *SamplingRule) MarshalJSON() ([]byte, error) {
+	s := struct {
+		Service string  `json:"service"`
+		Name    string  `json:"name"`
+		Rate    float64 `json:"sample_rate"`
+	}{}
+	if sr.exactService != "" {
+		s.Service = sr.exactService
+	} else if sr.Service != nil {
+		s.Service = fmt.Sprintf("%s", sr.Service)
+	}
+	if sr.exactName != "" {
+		s.Name = sr.exactName
+	} else if sr.Name != nil {
+		fmt.Printf("NAME: %#v, %v, %s\n", sr.Name, sr.Name, sr.Name)
```

Comment:
Can you do a final cleanup? When you feel it's ready for review, just ask for one from me.

Reply:
👍